src/helpers/assets.js

```diff
@@ -184,7 +184,7 @@ export const buildUniqueTokenList = (uniqueTokens, selectedShowcaseTokens) => {
   const showcaseTokens = [];
   const bundledShowcaseTokens = [];
 
-  const grouped = groupBy(uniqueTokens, token => token.collection.name);
+  const grouped = groupBy(uniqueTokens, token => token.collection?.name);
```

Comment:
I think this was handled on a separate PR

Reply:
yea, sorry. Wanted to make it runnable for me

Reply:
```suggestion
  const grouped = groupBy(uniqueTokens, token => token.collection.name);
```